common/data/data.json

```diff
@@ -423,7 +423,7 @@
         "sticky": false
       },
       {
-        "item": "Gold Stripped Party Hat",
+        "item": "Gold Striped Party Hat",
```

Comment:
Also I just realized this should just be Party Hat

Reply:
I thought of making it more new year themed